packages/cashier/src/modules/cashier-onboarding/cashier-onboarding.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useHasSetCurrency } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { useHistory } from 'react-router';
+import { PageContainer } from '../../components/page-container';
+import { useCashierStore } from '../../stores/useCashierStores';
+import {
+    CashierOnboardingAccountIdentifierMessage,
+    CashierOnboardingCryptoCard,
+    CashierOnboardingFiatCard,
+    CashierOnboardingOnrampCard,
+    CashierOnboardingP2PCard,
+    CashierOnboardingPaymentAgentCard,
+    CashierOnboardingSideNotes,
+} from './components';
+
+const CashierOnboarding: React.FC = observer(() => {
+    const history = useHistory();
+    const { ui } = useStore();
+    const { general_store } = useCashierStore();
+    const { setIsCashierOnboarding } = general_store;
+    const { toggleSetCurrencyModal } = ui;
+    const has_set_currency = useHasSetCurrency();
+
+    useEffect(() => {
+        setIsCashierOnboarding(true);
+
+        return () => {
+            setIsCashierOnboarding(false);
+            if (!has_set_currency && window.location.pathname.includes('/cashier')) {
+                history.push('/');
+                toggleSetCurrencyModal();
+            }
+        };
```

Comment:
@nijil-deriv So should we still keep it? 🤔 
If it's only for QA then I guess they should change the flow they test the app to be more realistic like a real user instead of we handling it in the code 🤔 What do you think? 🤔